java-rest-service/accelerator.yaml

```diff
@@ -326,4 +316,63 @@ engine:
         - type: ReplaceText
           substitutions:
           - text: "rest-service-db"
-            with: "#artifactId"
\ No newline at end of file
+            with: "#artifactId"
+
+    # DATABASE.md
+    - merge:
+      - exclude: [ "DATABASE.md" ]
+      - condition: "#databaseType == 'mysql'"
+        include: [ "DATABASE.md" ]
+        chain:
+          - type: ReplaceText
+            regex:
+              pattern: "Postgres"
+              with: "'MySQL'"
+          - type: ReplaceText
+            regex:
+              pattern: "PostgreSQL"
+              with: "'MySQL'"
+          - type: ReplaceText
+            regex:
+              pattern: "postgres"
+              with: "'mysql'"
+          - type: ReplaceText
+            condition: "#databaseIntegrationTestType == 'in-memory'"
```

Comment:
`databaseIntegrationTestType` is used here, but this accelerator won't have access to that option, because it's not exposed when importing the fragments `spring-boot-mysql` or `spring-boot-postgresql`.

Reply:
it's in the `spring-boot-h2` fragment which is imported by this accelerator

Reply:
Although, looking a bit closer at the fragments used, I think they need some additional refactoring to better separate the various concerns they address.